server/src/routes/cards.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+var express = require("express");
+const res = require("express/lib/response");
+var router = express.Router();
+const cardController = require("../controller/cardController");
+
+router.get("/", async function (_req, res, _next) {
+	res.send(await cardController.listCards());
+});
+router.get("/:cardType", async function (req, res, _next) {
+	res.send(await cardController.listCards(req.params.cardType));
+});
+router.post("/new", async function (req, res, _next) {
```

Comment:
```suggestion
router.post("/", async function (req, res, _next) {
```

Não há a necessidade de usar uma nova rota para esse caso, você pode usar a mesma rota para métodos diferentes do HTTP